app/frontend/src/features/search/SearchResult.tsx

```diff
@@ -41,26 +55,36 @@ export default function SearchResult({ user }: { user: User.AsObject }) {
       <Card className={classes.card}>
         <CardActionArea>
           <CardContent>
-            <div className={classes.resultHeader}>
-              <Avatar user={user} />
-              <Container>
-                <Typography variant="h2">{user.name}</Typography>
-                <Typography variant="subtitle1">
+            <UserSummary user={user} />
+            <div className={classes.statusLabelWrapper}>
```

Comment:
Sorry for not being clear, but do put this div in `UserSummary`. That way you won't need the left margin, and the vertical margin will be more consistent.

Reply:
Ok, back in!

Reply:
How does it fit with what Darren said though?

> That's a good catch actually, the children of UserSummary aren't really meant to be a dumping ground for everything other than some supplementary text which still fits with the user summary (e.g. the hosting status text and a user's host/meetup status) tbh when I first modified it.

Reply:
Semantically it makes sense (to me) that the hosting status is part of the "user summary", and also it makes the alignment better.